src/test/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsPartiesMappingTest.java

```diff
@@ -295,12 +317,23 @@ void buildHearingPartiesDetailsJointParty(YesNo jointParty) throws InvalidMappin
     void getIndividualInterpreterLanguageWhenHearingOptionsNull() throws InvalidMappingException {
 
         String individualInterpreterLanguage = HearingsPartiesMapping.getIndividualInterpreterLanguage(
-            null, null, referenceData
+            null, null, referenceData, null
         );
 
         assertThat(individualInterpreterLanguage).isNull();
     }
 
+    @DisplayName("When HearingOption is Null and and adjournLanguage is provided return empty string")
```

Comment:
and and

Reply:
Not clear what this test is testing. The description doesn't seem to match the assertion.

Reply:
fixed